### PR TITLE
LPS-170970 Adding a temporary workaround to fix glitches in the CI routine

### PR DESCRIPTION
**Pull request by luanakarla**

### CODE REVIEW

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:osb-site-initializer-customer-portal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro [hunk 1]
```diff
@@ -143,6 +143,14 @@ definition {
 			value1 = "${value}");
 	}
 
+	// Workaround created because the LPS-168369
```
Comment (manoelcyreno) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro:146]: To keep the same pattern of our poshi code, can you replace it for:
"Workaround for LPS-168369"

**Review by manoelcyreno — COMMENTED** [review]
Looks good, you only need to change the comment to the same pattern that we have on the poshi code.

**Comment by luanakarla** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by luanakarla** [comment]
ci:forward

**Pull request closed without merging**